scripts/fix_security_group.py

```diff
@@ -0,0 +1,167 @@
+import argparse
+import logging
+import os
+
+import requests
```

Comment:
This does not come out of the box with python. Is there anything else we can use? Otherwise we will have to do a pip install (not ideal)

Reply:
Latest revision has removed the "requests" external dependency, and uses the native "urllib" instead. Now both python scripts are only using native imports.

Tested these new scripts in schematics, everything is working there as well.